packages/api/src/index.js

```diff
@@ -104,6 +105,11 @@ r.add('get', '/internal/tokens', withMode(tokensList, RO), [postCors])
 r.add('post', '/internal/tokens', withMode(tokensCreate, RW), [postCors])
 r.add('delete', '/internal/tokens', withMode(tokensDelete, RW), [postCors])
 
+// Blog
+r.add('post', '/internal/blog/subscribe', withMode(blogSubscribe, RW), [
+  postCors,
+])
```

Comment:
Doesn't need maintenance mode restrictions.

```suggestion
r.add('post', '/internal/blog/subscribe', blogSubscribe, [postCors])
```